config/routes.rb

```diff
@@ -1,56 +1,19 @@
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
 
-  # You can have the root of your site routed with "root"
-  # root 'welcome#index'
+  resources :questions
+  resources :users
+  resources :tags
```

Comment:
you should restrict the resource to only the routes you need. index, show in this case.